.baseline/copyright/999_palantir.txt

```diff
@@ -0,0 +1 @@
+Copyright ${today.year} Palantir Technologies, Inc. All rights reserved.
```

Comment:
sigh, this shouldn't be created for external repos

Reply:
this PR should have been auto-updated, but we're working out some kinks with Excavator OSS. See internal issue devtools/excavator#502